lib/types.h

```diff
@@ -49,6 +49,7 @@ struct CodegenError : public std::runtime_error {
 namespace attributes {
     constexpr const char Annotation[] = "annotation";
     constexpr const char Color[] = "color";
+    constexpr const char DateTime[] = "dateTime";
```

Comment:
As written in the other PR, all of the attributes are only lowercase and i think it would be better to have this as `datetime` instead for consistency